### PR TITLE
{bio}[GCCcore/8.2.0] FragGeneScanPlus v20161116

### PR DESCRIPTION
**Pull request by agimenog**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/hallamlab',
+        'repo_name': 'FragGeneScanPlus',
+        'commit': '91b0ab6',
+    },
+}]
+checksums = ['53d88b435eeb6b32e64a7be189a8903fa23bcb22694d7a322d564490e9cd3312']
+
```
Comment (casparvl) [inline easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb:27]: As the bot indicates, this should have a `binutils` build dependency.

**Review by casparvl — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb (hunk 1 above)
Comment (agimenog) [reply easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb:29]: hi @casparvl i already fixed that! can you check? thanks

**Comment by casparvl** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/143941942903fd92f7159c19088e7cf7 for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
tcn1.local.snellius.surf.nl - Linux RHEL 8.6, x86_64, AMD EPYC 7H12 64-Core Processor, Python 3.6.8
See https://gist.github.com/casparvl/46b7eb2c1f9da2c602e942309627d785 for a full test report.

File: easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb (hunk 1 above)
Comment (casparvl) [reply easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb:29]: This part is fine, CI passes now

File: easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+homepage = 'https://github.com/hallamlab/FragGeneScanPlus'
+description = """FragGeneScan-Plus for scalable high-throughput short-read open reading frame prediction"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+builddependencies = [('binutils', '2.31.1')]
+
+sources = [{
+    'filename': '%(name)s-%(version)s.tar.gz',
+    'git_config': {
```
Comment (casparvl) [inline easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb:22]: Using `git_config` triggers a git clone by EasyBuild, which then tars the sources. That leads to non-deterministic checksums. 

You can actually get this same source at https://github.com/hallamlab/FragGeneScanPlus/archive/91b0ab6.tar.gz , which I believe _should_ give you a reproducible commit. See e.g. https://github.com/easybuilders/easybuild-easyconfigs/blob/4432f2762c94ee556f58c7fe8ab28d3f286b466b/easybuild/easyconfigs/p/pfind/pfind-20220613-gompi-2021b.eb#L5

Even more elegant than that example: I think you can specify
```
source_urls = GITHUB_SOURCE
github_account=hallamlab
sources = [local_commit + '.tar.gz']
```
but you'd have to give that a try (see https://docs.easybuild.io/version-specific/easyconfig-templates/#template-constants-that-can-be-used-in-easyconfigs fro the use of templates in EasyConfigs).

Comment (casparvl) [reply easybuild/easyconfigs/f/FragGeneScanPlus/FragGeneScanPlus-20161116-GCCcore-8.2.0.eb:22]: Yep, better example: https://github.com/easybuilders/easybuild-easyconfigs/blob/4432f2762c94ee556f58c7fe8ab28d3f286b466b/easybuild/easyconfigs/m/MARS/MARS-20191101-GCCcore-8.3.0.eb#L7

**Review by casparvl — CHANGES_REQUESTED** [review]

**Pull request closed without merging**